package.json

```diff
@@ -37,8 +37,7 @@
     "typecheck": "tsc -p ."
   },
   "dependencies": {
-    "@typescript-eslint/experimental-utils": "^2.5.0",
-    "micromatch": "^4.0.2"
```

Comment:
Don't forget to remove `@types/micromatch`, and commit the updated `yarn.lock` ;)